edit-post/components/header/keyboard-shortcuts-toggle/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress Dependencies
+ */
+import { withSelect, withDispatch } from '@wordpress/data';
+import { displayShortcut } from '@wordpress/keycodes';
+
+/**
+ * WordPress Dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/compose';
+import { MenuItem } from '@wordpress/components';
+
+const modalName = 'edit-post/keyboard-shortcut-help';
+
+export function KeyboardShortcutsToggle( { toggleModal, isModalActive } ) {
+	return (
+		<MenuItem
+			icon={ isModalActive && 'yes' }
+			isSelected={ isModalActive }
+			onClick={ toggleModal }
+			shortcut={ displayShortcut.access( 'h' ) }
+		>
+			{ __( 'Keyboard Shortcuts' ) }
```

Comment:
Looks a little strange that this copy wraps onto two lines. Could we widen the menu or shorten the text?

Also, the shortcut text on the right isn't vertically centred.

<img width="277" alt="screen shot 2018-08-09 at 12 10 17" src="https://user-images.githubusercontent.com/612155/43874101-36315e14-9bcd-11e8-93f8-c7cf9d2fbc89.png">

Reply:
Yeah, I spotted that. I wonder if shortening Option to Opt might be something we'd consider. Or using the symbol. We might still have this issue with translations, so worth making it look as nice as possible when wrapped.

@jasmussen any opinions on this?

Reply:
> We might still have this issue with translations.

Yeah, let's definitely vertically centre the text on the right regardless of what we do here.

Reply:
This is what it looks like with the option symbol:
![screen shot 2018-08-09 at 3 35 15 pm](https://user-images.githubusercontent.com/677833/43884887-e989a874-9be9-11e8-8dd7-94f56b351863.png)

Reply:
If we wanted to go crazy, could also use ⇧ for shift.

Reply:
It looks like https://github.com/WordPress/gutenberg/pull/8756 might solve this 🙂